lib/state.js

```diff
@@ -437,9 +454,8 @@ StateResolver.prototype.resolveMirrorFast_ = function(mirror) {
 
 StateResolver.prototype.getMirrorProperties_ = function(mirror) {
   var numProperties = this.config_.capture.maxProperties;
-  var namedProperties = mirror.properties(1, numProperties);
-  var indexedProperties = mirror.properties(2, numProperties);
-  return namedProperties.concat(indexedProperties);
+  var properties = mirror.properties();
```

Comment:
This does work as it is only executed on newer versions of node already. The culprit was the call to  `propertyNames` above. Fixed.